### PR TITLE
test: add disk type tests

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
add more disk type tests

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: test/e2e/testsuites/testsuites.go [hunk 1]
```diff
@@ -432,7 +432,7 @@ func NewTestDeployment(c clientset.Interface, ns *v1.Namespace, command string,
 		testDeployment.deployment.Spec.Template.Spec.NodeSelector = map[string]string{
 			"kubernetes.io/os": "windows",
 		}
-		testDeployment.deployment.Spec.Template.Spec.Containers[0].Image = "e2eteam/busybox:1.29"
```
Comment (andyzhangx) [inline test/e2e/testsuites/testsuites.go:435]: @claudiubelu I noticed there is an update on image `e2eteam/busybox:1.29` a day ago, it changed the behavior of this image, error is like following:
```
Pod azurefile-volume-tester-drv6b has the following logs: The argument 'echo $null >> C:\mnt\test-1\data' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Usage: pwsh[.exe] [[-File] <filePath> [args]]
                  [-Command { - | <script-block> [-args <arg-array>]
                                | <string> [<CommandParameters>] } ]
                  [-ConfigurationName <string>] [-CustomPipeName <string>]
                  [-EncodedCommand <Base64EncodedCommand>]
                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]
                  [-Interactive] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
                  [-OutputFormat {Text | XML}] [-Version] [-WindowStyle <style>]
                  [-WorkingDirectory <directoryPath>]

       pwsh[.exe] -h | -Help | -? | /?

PowerShell Online Help https://aka.ms/pscore6-docs

All parameters are case-insensitive.
```

which means `powershell.exe command` does not work on current `e2eteam/busybox:1.29`, do you know what's the correct command to fix this? I think I should add sth. in Line 437, e.g. 
```
testDeployment.deployment.Spec.Template.Spec.Containers[0].Args = []string{"-Command", command}
```

Comment (andyzhangx) [reply test/e2e/testsuites/testsuites.go:435]: so in this PR, I tried to revert back to use old windows image to workaround.

Comment (chewong) [reply test/e2e/testsuites/testsuites.go:435]: I can take a look at this issue

Comment (andyzhangx) [reply test/e2e/testsuites/testsuites.go:435]: cc @ZeroMagic 
let's merge this workaround fix first, thanks.

**Pull request merged**